src/mpyl/utilities/jenkins/runner.py

```diff
@@ -159,4 +160,5 @@ def run(self):
 
         new_build_number = last_build_number + 1
 
-        self.follow_logs(job, new_build_number, last_build)
+        if self.follow:
+            self.follow_logs(job, new_build_number, last_build)
```

Comment:
Perhaps there's value in a `silent mode` too? Where you block (showing progress bar) until the build is done, but don't stream the logs.

Not for this PR, but just a thought.

Reply:
I was doubting between the two approaches, if it should be silent or follow by default. But your version seems like a nice idea as well. Maybe let's hear what the users prefer as the default